doc/programming_guide/asr/deploying/native_windows.rst

```diff
@@ -4,7 +4,15 @@
 Deploying the Firmware with Native Windows
 ******************************************
```

Comment:
Good idea to have dedicated windows section to avoid repetition.
I also found out you can use setx from the CL to permanently change the env in windows
https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/setx